protocol/0081-SUCM-successor_markets.md

```diff
@@ -104,6 +106,14 @@ When fetching a market that is part of a "parent / successor chain", we should s
 
 After a LNL checkpoint restart the successor (child) / parent market state is preserved where applicable inc. the LPs ELS	(<a name="0081-SUCM-016" href="#0081-SUCM-016">0081-SUCM-016</a>)
 
+A market which has been settled, but is still inside successor expiry window, is retained in a checkpoint, and can be used by a successor market after restart(<a name="0081-SUCM-029" href="#0081-SUCM-029">0081-SUCM-029</a>)
+
+A market which has been settled, and beyond the successor expiry window, is not retained in a checkpoint, and cannot be used by a successor market after restart(<a name="0081-SUCM-030" href="#0081-SUCM-030">0081-SUCM-030</a>)
+
+A market which has been settled, and already has a child which has succeeded it, is retained in a checkpoint, and can be found after restart. Both child and parent retain child/parent links in the API(<a name="0081-SUCM-031" href="#0081-SUCM-031">0081-SUCM-031</a>)
+
+For a parent/child (explicitly: the child has left opening auction), after a checkpoint restart, parent/child enter opening auction again. It is not possible to propose a new market which attempts to succeed that parent.(<a name="0081-SUCM-032" href="#0081-SUCM-032">0081-SUCM-032</a>)
```

Comment:
Both parent and child enter opening auction again?

Reply:
Yes, this is technically possible (more so than something that would happen in real life), but given that it's possible for a successor to leave opening auction before the parent is settled, this is a possibility. We could have a checkpoint taken at a point in time where the parent market wasn't settled yet, and the successor has already left opening auction. If that checkpoint is then used to restore state from while the parent is still valid, both markets will be restored in opening auction, but because we've already copied over ELS data from the parent to the successor, it ought to not be possible for a second successor market for that parent